RadixWallet/Clients/DeepLinkHandlerClient/DeepLinkHandlerClient+Live.swift

```diff
@@ -0,0 +1,39 @@
+extension DeepLinkHandlerClient {
+	public static var liveValue: DeepLinkHandlerClient {
+		@Dependency(\.radixConnectClient) var radixConnectClient
+		@Dependency(\.errorQueue) var errorQueue
+
+		struct State {
+			/// The deepLink is buffered so it can be handled at the appropriate time based on the app state.
+			/// For example, if the deepLink is received while user is in onboarding flow,
+			/// we would want to handle it only when user did land on home screen.
+			var bufferedDeepLink: URL?
+		}
+
+		var state = State()
```

Comment:
Im unsure if this passes complete Isolation check? Since `state` is references referenced/mutated from multiple closures. 

Might be solved with `@TaskLocal`?

Reply:
but all of the closures are not executed in a concurrent context, there will not be two concurrent deepLinks to be handled at the sam time.